<launch> 
    <arg name="visual" default="false"/>

    <!-- moos ros bridge -->
    <arg name="wamv_factor" default="bamboo_wamv"/>
    <arg name="sim" default="false"/>
    <arg name="moos_folder" default="wamv_2014_task1"/>

    <!-- MOOS ROS Bridge -->
    <node pkg="moosros" type="Bridge" name="Bridge" 
        args="$(find moosros)/missions/$(arg moos_folder)/vehicle.xml $(find moosros)/missions/$(arg moos_folder)//wamv_vehicle.moos" 
        respawn="false" output="screen" />

    <remap from="moos_waypt/obj_list" to="/obj_list/odom"/>    
    <include file="$(find moos_bhv)/launch/moos_waypt.launch">
        <arg name="sim" value="$(arg sim)"/>
        <arg name="wamv_factor" value="$(arg wamv_factor)"/>
    </include>          

</launch>
